src/entities.rs

```diff
@@ -464,7 +477,12 @@ impl Entities {
             init(id, &mut self.meta[id as usize].location);
         }
     }
-
+    // Since nonzero ops are not const yet (https://github.com/rust-lang/rust/issues/84186)
+    // unsafe is required to bypass the compiler error
+    /// Returns `NonZeroU32` from given number
+    pub(crate) const fn ret_non_zero_u32(num: u32) -> NonZeroU32 {
+        unsafe { NonZeroU32::new_unchecked(num) }
+    }
```

Comment:
This should be dropped in favor of invoking `NonZeroU32::new_unchecked` directly; it's not saving a significant amount of verbosity and it is a major safety hazard.